src/coreclr/vm/ceeload.cpp

```diff
@@ -3284,7 +3264,7 @@ Module::GetAssemblyIfLoaded(
                 }
 
                 if (pDomainAssembly && pDomainAssembly->IsLoaded())
-                    pAssembly = pDomainAssembly->GetCurrentAssembly(); // <NOTE> Do not use GetAssembly - that may force the completion of a load
+                    pAssembly = pDomainAssembly->GetAssembly(); // <NOTE> Do not use GetAssembly - that may force the completion of a load
```

Comment:
The comment now contradicts the code... ?

Reply:
Right. I think the comment was not correct for quite a while. There were no sideeffects with `GetCurrentAssembly` either. Perhaps some time ago there was a distinction.